mail-triggered.js

```diff
@@ -0,0 +1,24 @@
+require('dotenv').config();
+const sgMail = require('@sendgrid/mail');
+const { VERIFY_EMAIL } = require('./constants')
+
+sgMail.setApiKey(process.env.SENDGRID_API_KEY);
+const API_ROOT = process.env.API_ROOT_URL
+
+const sendVerificationEmail = async (email, newToken) => {
+  try {
+    await sgMail.send({
+      to: email, // Change to your recipient
+      from: 'info@tomaspeterec.sk', // Change to your verified sender
```

Comment:
I would remove these comments:
```suggestion
      to: email,
      from: 'info@tomaspeterec.sk',
```